doc/changes/changes_18.0.0.md

```diff
@@ -0,0 +1,17 @@
+# Common Module of Exasol Virtual Schemas Adapters 18.0.0, released 2025-03-??
```

Comment:
Don't forget to update the date, in case you want to release yet this week

Reply:
There will be a couple more PRs until I can release.